src/components/Canary/TristateLabelStandalone/index.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { TristateToggle } from "../../TristateToggle";
+import { updateParams, decodeUrlSearchParams } from "./../url";
+import { getConciseLabelState } from "./../labels";
+
+const TristateLabelStandalone = ({ label, className, labelClass, ...rest }) => {
+  const { labels: urlLabelState = {} } = decodeUrlSearchParams(
+    window.location.search
```

Comment:
Could we use react-router [useSearchParams](https://reactrouter.com/en/main/hooks/use-search-params) for url params?